playbooks/roles/test-client/tasks/main.yml

```diff
@@ -93,3 +93,7 @@
 - name: "Run test scripts"
   command: "{{ item.cmd }}"
   with_items: "{{ test_scripts }}"
+
+# Run the Wireguard tests
+- include: wireguard.yml
```

Comment:
This syntax will apply the tag to every task in the included file which could eliminate some repetition:

```
- include: wireguard.yml
  tags:
    - wireguard
```